docs/changes.rst

```diff
@@ -47,8 +48,8 @@ Released: not yet
     NotOpenSSLWarning: urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the
            'ssl' module is compiled with 'LibreSSL 2.8.3'
 
-  Also, the default set of cyphers available has changed to use the list of
-  ciphers configured by the system rather than ciphers defined by urllib3.
+  Also, the default set of cyphers available in OpenSSL 1.1.1 has changed to use the 
+  list of ciphers configured by the system rather than ciphers defined by urllib3.
```

Comment:
Can we please shorten this change log entry? It does not need to be a repetition of the trouble shooting section.

Reply:
done